tasks/src/table-question-answering/data.ts

```diff
@@ -0,0 +1,60 @@
+import type { Task DataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{ 
+			description:"The Fewshot Table dataset consists of tables that naturally occur on the web, that are formatted as few-shot tasks for fine-tuning language models to improve their few-shot performance.",
+			id: "JeremyAlain/123_test",
+		},
+	],
+	demo: {
+		inputs: [
+			{ label: "Table",
+		          filename: "tableQA.jpg",
+			  type: "img",
+			 },
+			 
+			 { label: "Question",
+		           content: "what is the number of reigns for Harley Race",
+			   type: "text",
+
+			 },
+		],
+		outputs: [
+			{ label: "Result",
+			  content: "7",
+			  type: "text",
+			},
+		],
+	},
+	metrics: 
+		[
+		{ description: "",
```

Comment:
Can you add a description? Same with other one.

Reply:
I'm googling for definitions of these metrics, but I can't find resources.

Reply:
I've just changed the content. Please check it.
I see they use "denotation accuracy" in the TAPEX paper.